guides/common/assembly_installing-satellite-server-connected.adoc

```diff
@@ -0,0 +1,86 @@
+[id="installing-satellite-server-connected"]
+= Installing {ProjectServer}
+
+ifeval::["{build}" == "satellite"]
+When you install {ProjectServer} from a connected network, you can obtain packages and receive updates directly from the Red Hat Content Delivery Network.
+
+Use the following procedures to install {ProjectServer}, perform the initial configuration, and import subscription manifests.
+
+ifeval::["{build}" != "foreman-deb"]
+Before you continue, consider which manifests are relevant for your environment. For more information on manifests, see link:{BaseURL}content_management_guide/managing_subscriptions/[Managing Subscriptions] in the _Content Management Guide_.
+endif::[]
+
+
+NOTE: You cannot register {ProjectServer} to itself.
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+Use the following procedures to install {ProjectServer} and perform the initial configuration.
```

Comment:
This sentence is included twice, once in `ifeval::["{build}" == "satellite"]` on line 7, then here in `ifeval::["{build}" != "satellite"]`. Can you put it out from ifevals for simplicity?

Reply:
They are slightly different sentences (the first one includes "and import subscription manifests"), but actually the second one should only be != foreman-deb so I do need to adjust it.

Reply:
I ended up moving the ifevals around into a more manageable order. There were so many "ifs" for manifests in Foreman (katello + red hat OS) that I decided to just not mention it in the intro for the foreman build as it seemed like overkill.